cmd/run.go

```diff
@@ -130,6 +140,13 @@ func (c *cmdRun) run(cmd *cobra.Command, args []string) error {
 				}
 			}
 		}()
+		go func() {
+			defer apiWG.Done()
+			<-srvCtx.Done()
+			if aerr := srv.Close(); aerr != nil {
```

Comment:
:thinking: Why directly Close and not a graceful Shutdown?

Reply:
Resolved by the [latest force-push](https://github.com/grafana/k6/compare/2a9644af7538430ede422fdfdf44ebd2da74c4d4..eea76b15afa365ec8e95369cec3ccbd801ed7f2a)?

Reply:
:+1: